game/game.cpp

```diff
@@ -36,6 +36,9 @@ void Game::game_loop() {
             _player->action(key, time, _game_region_width, _game_region_height);
             get_player_pos_for_view(_player->get_pos()); 
         }
+        else {
+            _player->stay(key);
+        }
 
```

Comment:
I don't like it.
Try to add this code into Player::action
Mb change parameters of function or something else

Reply:
It's the problem of event logic. I'm coming in player->action when event.type == KeyPressed. No matter which key, if it's pressed I do some actions. It's global event, and I can't add this logic into player::action. Either key is pressed, or not.

Reply:
It means, that you have to coming in action always, because action doesn't mean 'move'. Action also assumes hit, move, take, and others action of players.

Reply:
I'll think about it